pcli/src/command/tx.rs

```diff
@@ -169,10 +169,23 @@ impl TxCmd {
                     tokio::time::sleep(std::time::Duration::from_secs(6)).await;
                 }
             },
-            TxCmd::Swap { input, into, .. } => {
-                let _input = input.parse::<Value>()?;
-                let _into = asset::REGISTRY.parse_unit(into.as_str());
-                println!("Sorry, this command is not yet implemented");
+            TxCmd::Swap {
+                input,
+                into,
+                fee,
+                source,
+            } => {
+                let input = input.parse::<Value>()?;
+                // TODO: should we swap with specific units or the base denom only?
+                // This is unclear until the dex is further along.
+                let into = asset::REGISTRY.parse_unit(into.as_str()).base();
```

Comment:
I think this change (grabbing the base denom) makes it have the correct behavior -- if I specify `--into penumbra`, for instance, I'll end up with a note recording `upenumbra`, the base denom, which is what I want.